src/types/convert.js

```diff
@@ -44,7 +44,7 @@ export function hexadecimalToBinaryString (str) {
     while (bin.length < 4) {
       bin = '0' + bin
     }
-    binaryStr += bin
+    binaryStr += strReverse(bin)
```

Comment:
This is seemingly incorrect; you need to reverse order of hex digits within each byte as well.

Reply:
Fixed, i hope)